### ISSUE TITLE
Can completely bypass orderbook user registration, agreement to the terms and jurisdiction restrictions!

### ISSUE THREAD

**Issue by jaymansfield** [Polkadex-Substrate/Polkadex-Open-Beta#441]
There is currently no requirement at the pallet level to register and validate an email address, or agree to the polkadex terms and conditions, or even comply with the restricted jurisdictions! 

The user can call all the extrinsic's themselves without using the Polkadex orderbook frontend and skip the entire email/password setup.

For example, I submitted these directly inside the Polkadot.js developer tools:

![image](https://user-images.githubusercontent.com/6218678/205147876-f4b6d23e-6b34-4027-b149-f37bfdc58f0f.png)

So without using orderbook.polkadex.trade I was able to register a funding account, add a proxy and deposit funds. All using polkadot.js developer extrinsic's and all without registering an email address anywhere.

And then confirmed through the orderbook's graphql API that the account exists and has funds:

Account exists:
![image](https://user-images.githubusercontent.com/6218678/205154927-c4671ebd-9c0d-4a14-a885-548422430f34.png)

Balance exists:
![image](https://user-images.githubusercontent.com/6218678/205155006-b9727705-a7d8-4153-a741-3aaec25a7d12.png)

I could then load the proxy account into hummingbot and trade. There is no requirement for a email address and password, or to even open https://orderbook.polkadex.trade once.

onchainaddress:{esm99RRxKvainDsRR6UHRcju4vNnAw1ouZEw64Yfn12tc6HpQ}

**Comment by jaymansfield** [comment]
![image](https://user-images.githubusercontent.com/6218678/205158817-99f8f143-8acf-4b70-851f-1ffe976508e0.png)

![image](https://user-images.githubusercontent.com/6218678/205158857-59ac9db3-3612-4451-ac61-be84e09838f3.png)

![image](https://user-images.githubusercontent.com/6218678/205159010-f90ab784-9576-4a89-a033-1b98de0ab0ca.png)

**Comment by nuel77** [comment]
yep... the blockchain extrinsics are not blocked by any region or country. But we will be soon adding some more checks on the trading request to check for registered account before trading.

**Comment by nuel77** [comment]
Note: This issue is related to the unique account idenitfier problem for THEA.
